### PR TITLE
HTCONDOR-2009 vm-gahp-test

### PR DESCRIPTION
**Pull request by JaimeFrey**
<Insert PR description here, and leave checklist below for code review.>

# HTCondor Pull Request Checklist for internal reviewers

- [x] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [x] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [x] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [x] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [x] Verify that the branch destination of the PR matches the target version of the ticket
- [ ] Check for correctness of change
- [x] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [x] Check for documentation, if needed  (documentation [build logs](https://readthedocs.org/projects/htcondor/builds/))
- [x] Check for version history, if needed
- [x] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [x] Check that each commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

File: src/condor_vm-gahp/xen_type.linux.cpp [hunk 1]
```diff
@@ -1273,6 +1282,16 @@ XenType::checkXenParams(VMGahpConfig* config)
 				fixedvalue.c_str());
 		return false;
 	}
+
+	virConnectPtr libvirt_connection = virConnectOpen("xen:///");
```
Comment (Todd-L-Miller) [inline src/condor_vm-gahp/xen_type.linux.cpp:1286]: https://libvirt.org/uri.html seems to indicate that this should be `xen:///system`?

**Review by Todd-L-Miller — APPROVED** [review]
Approved pending test successful test completion and confirmation that `xen:///` works in practice if not in theory.

File: src/condor_vm-gahp/xen_type.linux.cpp (hunk 1 above)
Comment (JaimeFrey) [reply src/condor_vm-gahp/xen_type.linux.cpp:1286]: I concur with your assessment. `xen:///` is what the vm-gahp uses elsewhere in the code. I tested kvm, and xen doesn't appear to work on the CHTC machines. I want to say leave it alone unless someone complains, which hopefully own't happen before we decide to rip all of this code out.

Comment (Todd-L-Miller) [reply src/condor_vm-gahp/xen_type.linux.cpp:1286]: Better to be consistent with the rest of the code; if it's broken, that way at least it'll be broken in the same way (that is, the test will be for what we're actually doing) and that will make it easier to fix everywhere if we need to.

**Pull request merged**